### PR TITLE
Add extended API functions

### PR DESCRIPTION
**Pull request by PhILoXs**
**When merged this pull request will:**
- Implement #68 

### Added the following API functions
- getAllMarkers
- getMarkerConfig
- getMarkerScale
- getMarkerPos
- getPlayerUID
- setMarkerScale
- setMarkerScaleLocal

### CODE REVIEW

File: addons/markers/functions/fnc_setMarkerScaleLocal.sqf [hunk 1]
```diff
@@ -0,0 +1,40 @@
+#include "script_component.hpp"
+/**
+ *  Author: PhILoX
+ *
+ *  Description:
+ *      Sets the new marker scale. (Same as vanilla setMarkerSizeLocal)
```
Comment (Timi007) [inline addons/markers/functions/fnc_setMarkerScaleLocal.sqf:6]:
```suggestion
 *      Sets the scale/size of marker. (Same as vanilla setMarkerSizeLocal)
```

**Review by Timi007 — CHANGES_REQUESTED** [review]

File: addons/markers/functions/fnc_getPlayerUID.sqf
[the remark this answers is not in the document]
Comment (PhILoXs) [reply addons/markers/functions/fnc_getPlayerUID.sqf:1]: as of generateUniquePrefix playerUID is used

File: addons/markers/functions/fnc_getPlayerUID.sqf [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+ *      Returns PlayerUID from marker prefix.
+ *
+ *  Parameter(s):
+ *      0: STRING - Marker prefix.
+ *
+ *  Returns:
+ *      STRING - PlayerUID or -1 if it isn't an editale Metis marker.
+ *
+ *  Example:
+ *      ["mtsmarker#123/0/1"] call mts_markers_fnc_getPlayerUID
```
[the remark this answers is not in the document]
Comment (PhILoXs) [reply addons/markers/functions/fnc_getPlayerUID.sqf:15]: as of generateUniquePrefix playerUID is used

File: addons/markers/functions/fnc_getPlayerUID.sqf [hunk 3]
```diff
@@ -0,0 +1,24 @@
+#include "script_component.hpp"
+/**
+ *  Author: PhILoX
+ *
+ *  Description:
+ *      Returns PlayerUID from marker prefix.
```
[the remark this answers is not in the document]
Comment (PhILoXs) [reply addons/markers/functions/fnc_getPlayerUID.sqf:6]: as of generateUniquePrefix playerUID is used

File: addons/markers/functions/fnc_getMarkerPos.sqf [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+ *  Example:
+ *      ["mtsmarker#123/0/1"] call mts_markers_fnc_getMarkerPos
+ *
+ */
+
+params [["_namePrefix", "", [""]], ["_preserveElevation", false, [false]]];
+
+CHECKRET(_namePrefix isEqualTo "", ERROR("No marker prefix"));
+
+getMarkerPos [format["%1_frame", _namePrefix], _preserveElevation];
```
[the remark this answers is not in the document]
Comment (PhILoXs) [reply addons/markers/functions/fnc_getMarkerPos.sqf:24]: [0,0,0] is returned if marker does not exists

File: addons/markers/functions/fnc_getPlayerUID.sqf [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+ *
+ *  Parameter(s):
+ *      0: STRING - Marker prefix.
+ *
+ *  Returns:
+ *      STRING - PlayerUID or -1 if it isn't an editale Metis marker.
```
[the remark this answers is not in the document]
Comment (PhILoXs) [reply addons/markers/functions/fnc_getPlayerUID.sqf]: as of generateUniquePrefix playerUID is used
spelling and quotes are corrected

File: addons/markers/functions/fnc_getMarkerScale.sqf [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+ *  Example:
+ *      ["mtsmarker#123/0/1"] call mts_markers_fnc_getMarkerSize
+ *
+ */
+
+params [["_namePrefix", "", [""]]];
+
+CHECKRET(_namePrefix isEqualTo "", ERROR("No marker prefix"));
+
+(GVAR(namespace) getVariable _namePrefix) param [3]
```
[the remark this answers is not in the document]
Comment (PhILoXs) [reply addons/markers/functions/fnc_getMarkerScale.sqf]: The default return is [0,0], but we are working only with one scale number, so the default behavior is 0. This behavior is now documented inline.

File: addons/markers/functions/fnc_getMarkerConfig.sqf [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+ *  Example:
+ *      ["mtsmarker#123/0/1"] call mts_markers_fnc_getMarkerConfig
+ *
+ */
+
+params [["_namePrefix", "", [""]]];
+
+CHECKRET(_namePrefix isEqualTo "", ERROR("No marker prefix"));
+
+(GVAR(namespace) getVariable _namePrefix) param [1]
```
Comment (Timi007) [inline addons/markers/functions/fnc_getMarkerConfig.sqf:36]:
```suggestion
(GVAR(namespace) getVariable [_namePrefix, []]) param [1, []]
```

File: addons/markers/functions/fnc_getPlayerUID.sqf [hunk 8]
```diff
@@ -0,0 +1,24 @@
...
+ *  Example:
+ *      ["mtsmarker#123/0/1"] call mts_markers_fnc_getPlayerUID
+ *
+ */
+
+params [["_namePrefix", "", [""]]];
+
+CHECKRET([_namePrefix] call FUNC(isMtsMarker) isNotEqualTo 1, QUOTE(-1));
+
+((_namePrefix splitString "/") param [1])
```
Comment (Timi007) [inline addons/markers/functions/fnc_getPlayerUID.sqf:23]:
```suggestion
(_namePrefix splitString "/") param [1, "-1"]
```

**Review by Timi007 — CHANGES_REQUESTED** [review]

**Review by Timi007 — DISMISSED** [review]

**Comment by Timi007** [comment]
Don't forget to document the new functions in the wiki.

**Review by Timi007 — APPROVED** [review]

**Pull request merged**